### ISSUE TITLE
Cheap Xbox 360 Wireless Controllers (Radio 2.4Ghz)

### ISSUE THREAD

**Issue by bretbenz84** [dankraus/padawan360#8]
Having bought a cheap controller that did not work, I discovered that they behave as if they are wired USB controllers with a simple radio in between. Thus it does not work the same way, but with simple edits and a change of the library it works like a charm. I'd like to contribute my version that works with these $20 controllers.

Change the include for xboxrecv.h to #include <XBOXUSB.h>

Change line 170 to XBOXUSB Xbox(&Usb);

Change line 242 to   if (!Xbox.Xbox360Connected) {

Any further xbox function call remove the , 0 from that line. Since it's not one of four controllers (default 0) and just a single USB controller this gets it to work.

**Comment by dankraus** [comment]
Nice find @bretbenz84 ! Would you like to open up a Pull Request to include an additional sketch with these changes? Please include information on the readme as well about this addition, which controllers it’s expected to work with as well

**Comment by bretbenz84** [comment]
Yes I would absolutely love to share my version. It took me 5 hours to figure it out. I can also update the readme.
